### PR TITLE
Changes for NodeTuning cluster capability

### PR DESCRIPTION
**Pull request by dagrayvid**
This PR updates the openshift/api dependency to bring in the new NodeTuning capability. 

This PR also adds NodeTuning CRDs to the set that can be ignored when cluster capabilities are set to None, in test/extended/cli/explain.go. An [https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-nod[…]ode-tuning-operator-master-e2e-aws-ovn/1603285604148187136](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-node-tuning-operator/524/pull-ci-openshift-cluster-node-tuning-operator-master-e2e-aws-ovn/1603285604148187136) can be seen in the pending [NTO PR#524](https://github.com/openshift/cluster-node-tuning-operator/pull/524).

See also the pending [CVO PR](https://github.com/openshift/cluster-version-operator/pull/883#issuecomment-1380958905) which depends on this change.

### CODE REVIEW

File: test/extended/cli/explain.go [hunk 1]
```diff
@@ -188,6 +188,14 @@ var (
 		{Group: "metal3.io", Version: "v1alpha1", Resource: "provisionings"},
 	}
 
+	nodeTuningTypes = []schema.GroupVersionResource{
+		{Group: "tuned.openshift.io", Version: "v1", Resource: "profiles"},
+		{Group: "tuned.openshift.io", Version: "v1", Resource: "tuneds"},
```
Comment (wking) [inline test/extended/cli/explain.go:193]: I think you want to drop these two entries from `baseCRDTypes` up around line 152.  Not clear to me why `performanceprofiles` aren't mentioned in the `master` tip, but :shrug:, I guess you're adding them now.

**Comment by dagrayvid** [comment]
I don't want to smash "/retest" unnecessarily and it looks like these tests are failing on other open PRs as well. 

@wking, @deads2k I think this PR is ready for review.

**Review by wking — APPROVED** [review]
Cluster-bot [run][1], via:

```
test e2e 4.13,openshift/origin#27657,openshift/cluster-version-operator#883,openshift/cluster-node-tuning-operator#524 aws,no-capabilities
```

gives:

```console
$ curl -s https://storage.googleapis.com/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1615473375369498624/build-log.txt | grep 'oc explain should contain proper spec'
started: 1/37/414 "[sig-cli] oc explain should contain proper spec+status for CRDs [apigroup:config.openshift.io] [Suite:openshift/conformance/parallel]"
passed: (27.1s) 2023-01-17T23:23:09 "[sig-cli] oc explain should contain proper spec+status for CRDs [apigroup:config.openshift.io] [Suite:openshift/conformance/parallel]"
```

/lgtm

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1615473375369498624

**Comment by dagrayvid** [comment]
/assign @bparees

As you are aware of the plans to make NTO optional

**Comment by dagrayvid** [comment]
The failing e2e-gcp- tests are related to the issues on build04, being tracked in [OHSS-18195](https://issues.redhat.com/browse/OHSS-18195)

**Comment by dagrayvid** [comment]
I'm seeing this same failure for ci/prow/e2e-gcp-ovn-upgrade in other pending PRs, it does not seem related to this change.

/unhold

**Comment by dagrayvid** [comment]
@bparees @wking, given that the e2e-gcp-ovn-upgrade failure seems unrelated to this change, what do you think about overriding e2e-gcp-ovn-upgrade for this PR? 

This was done for [the 1.26 rebase](https://github.com/openshift/kubernetes/pull/1432), and openshift/origin https://github.com/openshift/origin/pull/27679. Context in [this thread](https://redhat-internal.slack.com/archives/CCH60A77E/p1674850480150059) and [this bug](https://issues.redhat.com/browse/OCPBUGS-6796).

**Comment by bparees** [comment]
opened https://issues.redhat.com/browse/OCPBUGS-6796

/override ci/prow/e2e-gcp-ovn-upgrade

**Comment by dagrayvid** [comment]
The e2e-gcp-ovn failure is a known cluster-monitoring-operator issue: https://issues.redhat.com/browse/OCPBUGS-3923

/retest

**Pull request merged**